src/tester.rs

```diff
@@ -551,13 +551,7 @@ fn run_tests(
                                 }
                             }
                             Status::Signal => {
-                                if cfg!(unix) {
-                                    failure.status = Some("Did not exit due to signal".to_owned());
-                                } else {
-                                    failure.status = Some(
-                                        "Use of signal not supported on this platform".to_owned(),
-                                    )
-                                }
+                                failure.status = Some("Did not exit due to signal".to_owned());
```

Comment:
This could be parsed as "did not exit all, signal or not". I think we mean something like "Exit was not due to signal"?

Reply:
Fixed: 91772b5b2346a02919a49eabbac9f10c5e9f8c90